descriptors/ModuleDescriptor-template.json

```diff
@@ -762,6 +793,43 @@
         "organizations-storage.banking-account-types.item.delete"
       ]
     },
+    {
+      "permissionName" : "organizations-storage.settings.collection.get",
+      "displayName" : "banking_account_type collection get",
+      "description" : "Get a collection of banking_account_types"
+    },
+    {
+      "permissionName" : "organizations-storage.settings.item.post",
+      "displayName" : "banking_account_type post",
+      "description" : "Create a new banking_account_type"
+    },
+    {
+      "permissionName" : "organizations-storage.settings.item.get",
+      "displayName" : "banking_account_type get",
+      "description" : "Fetch a banking_account_type"
+    },
+    {
+      "permissionName" : "organizations-storage.settings.item.put",
+      "displayName" : "banking_account_type put",
+      "description" : "Update a banking_account_type"
+    },
+    {
+      "permissionName" : "organizations-storage.settings.item.delete",
+      "displayName" : "banking_account_type delete",
+      "description" : "Delete a banking_account_type"
+    },
+    {
+      "permissionName" : "organizations-storage.settings.all",
+      "displayName" : "banking_account_type all",
+      "description" : "All permissions for banking_account_type",
+      "subPermissions" : [
+        "organizations-storage.settings.collection.get",
+        "organizations-storage.settings.item.post",
```

Comment:
**Question**
Can we have post and delete privileges even API doesn't allow this?

Reply:
removed